frontend/packages/console-plugin-shared/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@console/plugin-shared",
```

Comment:
we will be publishing under openshift-console npm org
```suggestion
  "name": "@openshift-console/plugin-shared",
```

Reply:
done